app/views/emailTemplates/reminderEmail.ejs

```diff
@@ -131,7 +131,7 @@
                     <table border="0" cellpadding="0" cellspacing="0" width="100%" style="max-width: 600px;">
                         <tr>
                             <td bgcolor="#ffffff" align="left" style="padding: 20px 30px 40px 30px; color: #666666; font-family: 'Lato', Helvetica, Arial, sans-serif; font-size: 18px; font-weight: 400; line-height: 25px;">
-                                <p style="margin: 0;">Get ready and get pumped for your upcoming class! Make sure to join on time and have everything you need to have a successful and worthwhile session. And don't forget to have fun! 😃</p>
+                                <p style="margin: 0;">Get ready and get pumped for your upcoming <%= classInfo.class_name %> class! It starts at <%= classInfo.scheduled_time %> and ends at <%= classInfo.class_end_time %>. Make sure to join on time and have everything you need to have a successful and worthwhile session. And don't forget to have fun! 😃</p>
```

Comment:
Can you remove this emoji? I think google mail update its policy if there are emojis in email body, the email is highly likely to go to spam

Reply:
removed